docs/ops/comparison/GreaterEqual_1.md

```diff
@@ -23,7 +23,7 @@ o_{i} = a_{i} \geq b_{i}
   * **Range of values**:
     * *none* - no auto-broadcasting is allowed, all input shapes should match
     * *numpy* - numpy broadcasting rules, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md)
-    * *pdpd* - PaddlePaddle-style implicit broadcasting, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md)
+    * *paddle* - PaddlePaddle-style implicit broadcasting, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md)
```

Comment:
we have `BroadcastType::PDPD` in API

@slyalin why frontend is named differently? Looks like will have 2 entities: PDPD and paddle

Reply:
@ilya-lavrenov it looks like not released yet, so we can rename
2021.4.2 version
https://github.com/openvinotoolkit/openvino/blob/6c4462759e8974c0ce5b36ad22972a1c6ded76a8/docs/ops/logical/LogicalAnd_1.md

Reply:
but it looks like BroadcastType ::PDPD is already released...

Reply:
Broadcast/Autobroadcast changes were reverted.
I discussed it with Sergei and we decided to leave pdpd here.